wetterdienst/core/point_data.py

```diff
@@ -422,3 +595,209 @@ def _create_humanized_parameters_mapping(self) -> Dict[str, str]:
         hcnm = {parameter.value: parameter.name for parameter in self._parameter_base}
 
         return hcnm
+
+
+class PointDataStationsCore(PointDataCore):
+    """ Core for stations information of a source """
+
+    # Columns that should be contained within any stations information
+    _base_columns = (
+        Columns.STATION_ID.value,
+        Columns.FROM_DATE.value,
+        Columns.TO_DATE.value,
+        Columns.STATION_HEIGHT.value,
+        Columns.LATITUDE.value,
+        Columns.LONGITUDE.value,
+        Columns.STATION_NAME.value,
+        Columns.STATE.value,
+    )
+    # TODO: eventually this can be matched with the type coercion of station data to get
+    #  similar types of floats and strings
+    # Dtype mapping for stations
+    _dtype_mapping = {
+        Columns.STATION_ID.value: str,
+        Columns.STATION_HEIGHT.value: float,
+        DWDMetaColumns.LATITUDE.value: float,
+        DWDMetaColumns.LONGITUDE.value: float,
+        DWDMetaColumns.STATION_NAME.value: str,
+        DWDMetaColumns.STATE.value: str,
```

Comment:
While this file is now within the `core` namespace, it still has `DWDMetaColumns` here. Can we improve on this here and at other occurrences?